frontend/src/components/Volunteer/volunteer.jsx

```diff
@@ -0,0 +1,191 @@
+import React, { Component } from "react";
+import { Container, Row, Col } from "react-bootstrap";
+import "./volunteer.css";
+import SingleCarousel from "../SingleCarousel/index";
+
+class Volunteer extends Component {
+
+  constructor(props){
+      super(props)
+
+
+      this.state = {
+        role:'',
+        name:'',
+        email:'',
+        phone:'',
+        website:'',
+        message:''
+       }
+    }
+
+
+
+  onChange = (event) => {
```

Comment:
this function is not being used right now. It should be used to set the state onChange of the inputs